docs/integrations/destinations/exasol.md

```diff
@@ -0,0 +1,72 @@
+# Exasol
+
+Exasol is the in-memory database built for analytics.
+
+## Sync overview
+
+### Output schema
+
+Exasol tables become Airbyte Streams and Exasol columns become Airbyte Fields. Each table will contain 3 columns:
```

Comment:
> Each table will contain 3 columns:

Which table? Exasol table? Or is this about a table describing an import or export *process* or *event*?
Can we use a more specific term than just "table"?